msfits/MSFits/MSFitsOutput.cc

```diff
@@ -2132,10 +2131,25 @@ Bool MSFitsOutput::_writeSU(std::shared_ptr<FitsOutput> output, const Measuremen
                         }
                     }
                     if (sourceColumns->properMotion().isDefined(rownr)) {
-                        Vector<Double> pm =
-                                sourceColumns->properMotion()(rownr);
-                        *pmra = pm(0);
-                        *pmdec = pm(1);
+                        Vector<Double> pm = sourceColumns->properMotion()(rownr);
+			String unit="UNCALIB";
+			unit = sourceColumns->properMotion().keywordSet().asArrayString("QuantumUnits").tovector()[0];
+			unit.upcase();
+			if (unit=="RAD/S"){ // convert to deg/day
+			    *pmra = pm(0)*C::day/C::degree;
+			    *pmdec = pm(1)*C::day/C::degree;
+			}
+			else{
+			    *pmra = pm(0);
+			    *pmdec = pm(1);
+			    if (unit.at(0,5)!="DEG/D"){
+			       os << LogIO::WARN
+				  << "Proper motion for source in SOURCE table row #"<<rownr<<" has unfamiliar units: "
+				  << unit << " .\n Value in uv fits table AIPS SU needs to be checked."
+				  << LogIO::POST;
+			    }
+			}
```

Comment:
Why not use casacore's built-in unit conversion routines? There's probably a bug in the code below, if you want to use it please try it locally.
```suggestion
			ArrayQuantColumn<Double> pm = sourceColumns->properMotionQuant()(rownr);
			try {
			    *pmra = pm_quantity(0).getValue("deg/d", true);
			    *pmdec = pm_quantity(1).getValue("deg/d", true);
			} catch (const std::exception& e) {
			       os << LogIO::WARN
				  << "Proper motion for source in SOURCE table row #"<<rownr<<" has unfamiliar units: "
				  << unit << " .\n Value in uv fits table AIPS SU needs to be checked."
				  << LogIO::POST;
		             *pmra = pm(0).getValue();
			     *pmdec = pm(1).getValue();
			}
```

Reply:
Yes, I was also thinking about this but was not sure the casacore routines would do the right thing in all cases.
The only use-case I know of is the rad/s -> deg/day conversion.
But OK, l will try your code and, if it works, make a new commit. 
Thanks!